### PR TITLE
Time period info in dashboard + WoS coverage

### PR DESCRIPTION
**Pull request by agnepagne**
We have discussed including information about time period for the firstpage indicators and including WoS coverage on firstpage. Trying to combine those here.

We should probably also discuss, and decide, if we want to use the last period/year for the firstpage or the entire time period at least for individuals (there is brief discussion in #39).

The WoS coverage indicator here is for the entire period, and only covers the DiVA publication types "Article, peer review" and "Conference paper, peer review". I am not sure how general WoS coverage is usually reported at KTH, should more publication types be included?

### CODE REVIEW

File: inst/extdata/abm.Rmd [hunk 1]
```diff
@@ -6,10 +6,10 @@ output:
...
     favicon: kth-logo.png
 params:
-  unit_code: u1g9umtq
-  is_employee: TRUE
-  use_package_data: FALSE
-  embed_data: TRUE
+  unit_code: KTH
+  is_employee: FALSE
+  use_package_data: TRUE
+  embed_data: FALSE
```
Comment (mskyttner) [inline inst/extdata/abm.Rmd]: I think these are good default params

Comment (agnepagne) [reply inst/extdata/abm.Rmd]: Indeed, changed to Sigbritt for testing and forgot to change back. :)

Comment (fileunderwater) [reply inst/extdata/abm.Rmd]: Couldn't these defaults (at least the unit_code one) be taken from the new abm_config(), to try to collect defaults in one place? Apparently, the YAML-block can take R variables, as e.g. (taken from https://stackoverflow.com/questions/31861569/setting-document-title-in-rmarkdown-from-parameters):

---
output: html_document
params: 
    set_title: "My Title!"
title: "`r params$set_title`"
---

Comment (fileunderwater) [reply inst/extdata/abm.Rmd]: Couldn't these defaults be placed in the new abm_config() (at least unit_code) to have all the defaults in one place.

For instance, this seems to work, which I found in the YAML-docs (https://cran.r-project.org/web/packages/ymlthis/vignettes/yaml-overview.html):

    params:
      unit_code: !r bibliomatrix:::abm_config()$default_unit

@mskyttner @agnepagne

**Comment by fileunderwater** [comment]
Couldn't these defaults be placed in the new abm_config() (at least unit_code) to have all the defaults in one place.

For instance, this seems to work, which I found in the YAML-docs (https://cran.r-project.org/web/packages/ymlthis/vignettes/yaml-overview.html):

    params:
      unit_code: !r bibliomatrix:::abm_config()$default_unit

However, the current master needs to be merged into this branch for this to work (since abm_defaults() are missing here).

@mskyttner @agnepagne

**Comment by agnepagne** [comment]
Skulle vi kunna mergea det här då kanske?

**Pull request merged**